### ISSUE TITLE
Increase Blur Intensity?

### ISSUE THREAD

**Issue by JudahMantell** [mob-sakai/UIEffect#258]
Hi! This is a fantastic asset that has been working great for me.  But unfortunately the blur level isn't high enough for my needs.  Is it possible to increase the level of blur using the UIEffect component?

This asset seems to be the only solid method of blurring UI Images, and it's a shame that the blur isn't that intense.
Any help would be greatly appreciated.

Thank you!

**Comment by mob-sakai** [comment]
Sorry for my late reply.

UIEffect v5 is now released.
Please try it. :+1:
[🔄 Upgrading from v4 to v5](https://github.com/mob-sakai/UIEffect?tab=readme-ov-file#-upgrading-from-v4-to-v5)

**Comment by lukaslabaj** [comment]
Hello, Im using v5, and I would like to also increase blur effect. I was able to do it with changing package locally, I tried two options : 
a. change multiplier directly inside of shader : `const half2 blur = texel_size() * intensity * 2;`
b. increased _SamplingIntensity range from 0-1 to 0-5 but that will effect another sampling effect
Any change you will update v5 with some solution for that?
Thank you
